### DISCUSSION TITLE
raspotify device accessible via google assistant

### DISCUSSION THREAD

**Discussion by cfkmo** [dtcooper/raspotify#630] [Q&A]
Hej!
I have set up my raspotify so that I can choose my old amplifier via spotify app - works perfectly fine. But as I am also using google assistant to start streaming music from spotify, I wondered if it is possible that the assistant recognizes the raspotify devices or if there is any other idea, I am not aware of. 

In the end I would like to create kind of automation including home assistant to turn on the amplifier as soon as google assistant starts streaming music on this device. But the precondition is that google recognized my amplifier :/ 

Thanks in advance for any suggestion :)

**Comment by giggywithit** [comment]
Get anywhere with this?
I seem to remember that you use the name you give the speaker.
You may need to mess around with the naming, type, etc....
Never tried but would like to know
